src/strapi-api.ts

```diff
@@ -0,0 +1,58 @@
+import type { RespJson } from "./common";
+
+export class StrapiAPI {
+    token: string = "";
+    env: any = (typeof process !== "undefined") ? process.env : global
+
+    public async getProfile() {
+        return this.fetch("/users/profiles")
+    }
+
+    public async getPoints() {
+        return this.fetch("/users/points")
+    }
+
+    private async fetch(urlPath: string) {
+        if (!this.token) {
+            this.token = await this.getToken()
+        }
+
+        let url = this.env.STRAPIBASEURL + urlPath
+        const response = await fetch(url, {
+            method: 'GET',
+            headers: {
+                'cookie': this.token
+            }
+        })
+        const result = await response.json<RespJson>();
+        return result
+    }
+
+    private async getToken(): Promise<string> {
+
+        var username: string = await cron_worker.get("strapi_username") as string;
+        var password: string = await cron_worker.get("strapi_password") as string;
+        if (!username || !password) {
```

Comment:
Can we comment that this is for dev environment?

Reply:
added coment